CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 =========
 
+0.12.0 (to be released)
+-----------------------
+
+* The :ref:`testing framework <web-poet-testing>` now allows defining a
+  :ref:`custom item adapter <web-poet-testing-adapters>`.
```

Comment:
Should we mention https://github.com/scrapinghub/web-poet/commit/7f401c0767a86c14b4ff2647e9c224644a218c5d?

Reply:
https://github.com/scrapinghub/web-poet/pull/174/commits/47a11325ca7d9a4b2248480982bc5bc7fbb26a1c ? I’m not sure if we should go into more detail (i.e. specifically suggesting to edit the corresponding JSON files to update the field name).